### ISSUE TITLE
Does the code support CUDA 11 ?

### ISSUE THREAD

**Issue by CoreloneH** [rosinality/stylegan2-pytorch#256]

**Comment by AYUSH-ISHAN** [comment]
Yes, it supportd. I ran it on google colab with CUDA version 11.2  and with some changes in code, I ran it on Pytorch version 1.9.0 and CUDA version 11.

**Comment by AYUSH-ISHAN** [comment]
Yes, it supportd. I ran it on google colab with CUDA version 11.2  and with some changes in code, I ran it on Pytorch version 1.9.0 and CUDA version 11.

**Comment by mo666666** [comment]
Hi, AYUSH-ISHAN! Would you like to share the changes made in your code? I find this repo doesn't work well on my computer with CUDA==11.7, torch==1.8.0.

**Comment by AYUSH-ISHAN** [comment]
https://github.com/AYUSH-ISHAN/Style_GAN_2
Here, is the link to code. It had been more than a year. So, may be some dependencies would have changed.
